comics/comics/anleggsplassen.py

```diff
@@ -1,40 +1,43 @@
 import re
 
+import dateutil.parser
```

Comment:
`dateutil` is only a transitive dependency for now, so if you really need this, it must be added to the dependencies with:

```
poetry add dateutil
```